editor/src/components/inspector/common/inspector-end-to-end-tests.spec.browser2.tsx

```diff
@@ -2100,9 +2100,9 @@ describe('inspector tests with real metadata', () => {
           await setControlValue('padding-V', '20', renderResult.renderedDOM)
         },
         endSnippet: makeCodeSnippetWithKeyValue({
-          paddingLeft: 10,
-          paddingTop: 20,
-          paddingBottom: 20,
+          paddingLeft: 10, // no unit because it wasn't affected by the change
+          paddingTop: '20px',
```

Comment:
hmm, the standard Utopia behavior for single values is to omit the `px` when it is not needed. so these tests should remain `paddingTop 20,` I believe

Reply:
https://github.com/concrete-utopia/utopia/pull/3260/commits/279a6c1a6993456e01c55101c77d8df32a9ddfaa